apps/src/applab/commands.js

```diff
@@ -888,6 +888,36 @@ applabCommands.textInput = function (opts) {
   return Boolean(Applab.activeScreen().appendChild(newInput));
 };
 
+applabCommands.textArea = function (opts) {
+  console.log('textArea');
+  // PARAMNAME: textArea: id vs. areaId
+  apiValidateDomIdExistence(opts, 'textArea', 'id', opts.elementId, false);
```

Comment:
Too much work to refactor to share between textArea and textLabel?

Reply:
Based on your feedback below, implementation is now different from `textLabel`.